### DISCUSSION TITLE
Programming languages

### DISCUSSION THREAD

**Discussion by telmobss1** [lxqt/lxqt#1955] [General]
qt provides support for multiple programming languages
Rust python c++ pyqt
My goal with this topic is to know and clarify what programming languages are accepted by the lxqt team

**Comment by wxl** [comment]
C++. That's it. Nothing else.

**Reply by tsujan** [reply]
C++, definitely. Also, see https://github.com/lxqt/lxqt/issues/1433#issuecomment-516556436

I don't think LXQt has used pyqt or rust anywhere.
